core/src/test/scala/com/microsoft/azure/synapse/ml/explainers/split1/ICEExplainerSuite.scala

```diff
@@ -0,0 +1,154 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.explainers.split1
+
+import org.apache.spark.ml.{Pipeline, PipelineModel}
+import org.apache.spark.ml.feature.{OneHotEncoder, StringIndexer, VectorAssembler}
+import com.microsoft.azure.synapse.ml.core.test.base.TestBase
+import com.microsoft.azure.synapse.ml.core.test.fuzzing.{TestObject, TransformerFuzzing}
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.functions._
+import org.apache.spark.ml.classification.LogisticRegression
+import com.microsoft.azure.synapse.ml.explainers.{ICECategoricalFeature, ICENumericFeature, ICETransformer}
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.ml.util.MLReadable
+import scala.jdk.CollectionConverters._
+
+
+class ICEExplainerSuite extends TestBase with TransformerFuzzing[ICETransformer] {
+
+  import spark.implicits._
+  val dataDF: DataFrame = (1 to 100).flatMap(_ => Seq(
```

Comment:
please make all things in the test class body lazy, these will run during python test generation if they arent